components/integrations/installed_oauth_apps/installed_oauth_apps.jsx

```diff
@@ -7,7 +7,7 @@ import {FormattedMessage} from 'react-intl';
 
 import {localizeMessage} from 'utils/utils.jsx';
 import BackstageList from 'components/backstage/components/backstage_list.jsx';
-import InstalledOAuthApp from '../installed_oauth_app';
+import InstalledOAuthApp from '../installed_oauth_app/';
```

Comment:
Note here: if I don't put the trailing `/`, `eslint` complains about the following:

> Casing of ../installed_oauth_app does not match the underlying filesystem.

Reply:
I'm wondering if circle ci will also complain about this if you make a commit without the slash. If circle ci says it's fine, there may be an issue with your eslint setup. If circle ci complains about it, there's another problem to solve here.

Reply:
Don't you have the same issue in your ESLint setup? I am quite sure that my setup will only use what's in the project eslint config...

With that being said, I can easily try and see what happens in CircleCI...

Reply:
CI is passing, so that's strange if you're still getting an issue locally. I know the configuration of imports with ESLint vs other things is a bit painful, but usually relative paths are the thing that actually works well.

Your PR looks otherwise good to merge. I'd be totally fine taking it if it passes CI unless you still wanted to look at it. Otherwise, we can just hope it's an intermittent ESLint problem

Reply:
If all look good now and there are no other standing comments, I'd say "let's merge" :)